javascript/builtins/Map.json

```diff
@@ -0,0 +1,1060 @@
+{
+  "javascript": {
+    "builtins": {
+      "Map": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Map",
+          "support": {
+            "webview_android": {
+              "version_added": "38"
+            },
+            "chrome": {
+              "version_added": "38"
```

Comment:
So we don't mark the experimental Chrome flags?

Reply:
I wasn't sure if it is really worth it. Chrome 31 is quite old now and it adds complexity to the table. Happy to add it, if you think preserving this info is important, though.

Reply:
Fine without them for me.